my_list.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&)
+    @list.each(&)
+  end
```

Comment:
- [ ] I am unable to run the tests due to an error in this method. Kindly fix it.

  ![raid-1](https://user-images.githubusercontent.com/36542986/174054824-ad312181-ead6-4617-b634-81a95c40a952.jpg)